src/tokens.js

```diff
@@ -110,6 +114,29 @@ export const legacyPrint = new ExternalTokenizer(input => {
   }
 })
 
+function longStringContent(quote) {
+  return new ExternalTokenizer(input => {
+    const qord = quote.charCodeAt(0);
+    const token = quote === "'" ? longString1lContent : longString2lContent;
+    const testExit = quote === "'" ? /['\\]/ : /["\\]/;
+    let i = 0;
+    for(;; i++) {
+      // End of long string
+      const char = String.fromCharCode(input.next);
+      if (char === quote && input.peek(1) === qord && input.peek(2) === qord) {
+        break
+      }
+      if (testExit.test(char)) {
```

Comment:
this should probably not be a regex, as I'm reviewing it. I'll change it to `char === quote || char === '\\'` as well

Reply:
Resolved